pyGPG/gpg.py

```diff
@@ -111,6 +111,27 @@ def runGPG(self, task=None, inputtxt=None, inputfile=None, outputfile=None):
             return GPGResult(gpg, results, extract_stdout=True)
         return GPGResult(gpg, results)
 
+    def listkey(self, id_string=None, outputfile=None):
+        '''Lists the keys with --list-key <argument>
+
+        @param id_string: string (optional) argument to --list-key
```

Comment:
I know you followed the fingerprint() example, but I wonder if we should make id_string non-optional.  For both functions.  Then the same for fingerprint() and any others like it.